powder_completion.go

```
@@ -0,0 +1,216 @@
+	if p.PowderCommand != nil {
+		return p.PowderCommand(ctx, args...)
+	}
+	if !p.ResolveTools {
+		output, err := p.powder(ctx, args...)
+		return output, nil, err
+	}
+	path, err := trustedExecutable(p.Root, "powder")
+	if err != nil {
+		return nil, nil, err
+	}
+	command := exec.Command(path, args...)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)